scripts/setup-adapters.sh

```diff
@@ -33,32 +33,20 @@ function install_aws_deps {
   cmake_install -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DBUILD_SHARED_LIBS:BOOL=OFF -DMINIMIZE_SIZE:BOOL=ON -DENABLE_TESTING:BOOL=OFF -DBUILD_ONLY:STRING="s3;identity-management"
   # Dependencies for S3 testing
   # We need this specific version of Minio for testing.
-  if [[ "$OSTYPE" == linux-gnu* ]]; then
-    wget https://dl.min.io/server/minio/release/linux-amd64/archive/minio-20220526054841.0.0.x86_64.rpm
-    if yum list installed minio >/dev/null 2>&1;then
```

Comment:
The refactor unifies the mac and linux paths. A specific older Minio version is used for S3Testing. I think we should include that version name in the binary to avoid conflicts.
I will make this change.

Reply:
fixed.

Reply:
I moved the Minio changes to its own PR https://github.com/facebookincubator/velox/pull/10373